Essentials/src/com/earth2me/essentials/Settings.java

```diff
@@ -1623,4 +1623,9 @@ private double _getMaxProjectileSpeed() {
     public double getMaxProjectileSpeed() {
         return maxProjectileSpeed;
     }
+
+    @Override
+    public boolean isSpawnIfNoHome() {
+        return config.getBoolean("spawn-if-no-home", false);
```

Comment:
This should default to true to preserve the previous behaviour.

Reply:
Updated.